### PR TITLE
homepage layout and create post component

### PR DESCRIPTION
**Pull request by sashaboi**
Created homepage layout with two sidebars , a create post component and a feed. 
screencapture-localhost-3000-2022-05-12-02_54_56

functionality check : letter counter and button disabled at create post component.
Not a lot of functionality yet, but would love a review .
Yet to do : figure out what @bhavya said about jsconfig file to change imports style , will do in next pr.

### CODE REVIEW

File: src/pages/Auth/Login.jsx [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
-      <div className="center-items-container">Login</div>
+      <div className="center-items-container">
+        <div className="single-function-page-container">
+          <div className="single-page-heading">Login</div>
+          <hr />
+          <div className="inputs-container">
+            <label htmlFor="email">Email </label>
+            <input type="email" placeholder="adarshbalika@gmail.com" />
+            <label htmlFor="password">Password </label>
+            <input type="password" placeholder="adarshBalika123" />
```
Comment (devdeadviz) [inline src/pages/Auth/Login.jsx:17]: One suggestion, whenever you are using input fields or creating some type for form, then using the form element is a good practice.

File: src/pages/Homepage/homepage.css [hunk 2]
```diff
@@ -3,8 +3,11 @@
   flex-direction: column;
   justify-content: space-between;
 }
-.center-items-container {
+.home-page-container {
+  margin: auto;
+  text-align: center;
+  width: 80vw;
```
Comment (devdeadviz) [inline src/pages/Homepage/homepage.css:9]: Avoid using `viewport units` as it causes issues when you scale your project.

**Review by devdeadviz — APPROVED** [review]
The code is clean, your file structure is also good. I like the part where you have segregated every functionality and component.

One suggestion, you can use the `Navbar` and `Footer` components directly in the `App.js` file so as to avoid repetition of it as they will be used in every file.

**Review by harshitpaliwal95 — APPROVED** [review]
good file folder structure 
everything looks good

File: src/pages/Auth/Login.jsx (hunk 1 above)
Comment (sashaboi) [reply src/pages/Auth/Login.jsx:17]: I agree, I will change this in upcoming prs.

File: src/pages/Homepage/homepage.css (hunk 2 above)
Comment (sashaboi) [reply src/pages/Homepage/homepage.css:9]: yes.

**Pull request merged**